server/shared/src/test/scala/org/http4s/server/staticcontent/StaticContentShared.scala

```diff
@@ -20,38 +20,27 @@ package staticcontent
 
 import cats.effect.IO
 import fs2._
+import fs2.io.file.Files
+import fs2.io.file.Path
 import org.http4s.syntax.all._
 import org.http4s.testing.AutoCloseableResource
 
 import java.nio.charset.StandardCharsets
-import java.nio.file.Files
 import java.nio.file.Paths
 
 private[staticcontent] trait StaticContentShared { this: Http4sSuite =>
   def routes: HttpRoutes[IO]
 
-  lazy val testResource: Chunk[Byte] = {
-    val s = getClass.getResourceAsStream("/testresource.txt")
-    require(s != null, "Couldn't acquire resource!")
-    val bytes =
-      AutoCloseableResource.resource(
-        scala.io.Source
-          .fromInputStream(s)
-      )(
-        _.mkString
-          .getBytes(StandardCharsets.UTF_8)
-      )
+  protected val defaultSystemPath: String =
+    org.http4s.server.test.BuildInfo.test_resourceDirectory
+      .replace("jvm", "shared")
+      .replace("js", "shared")
 
-    Chunk.array(bytes)
-  }
+  lazy val testResource: IO[Chunk[Byte]] =
```

Comment:
tried to revert these to just `Chunk[Byte]`, but there's no `unsafeRunSync` in js, right?

Reply:
Yeah, no `unsafeRunSync`. That's fine, thanks for trying to minimize the changes :)